packages/suite/src/views/settings/device/FirmwareVersion.tsx

```diff
@@ -69,7 +68,10 @@ export const FirmwareVersion = ({ isDeviceLocked }: FirmwareVersionProps) => {
     const currentFwVersion = getFwVersion(device);
     const availableFwVersion = getFwUpdateVersion(device);
     const { revision } = device.features;
-    const githubUrl = GITHUB_FW_COMMIT_URL + revision;
+    const githubUrl = `https://github.com/trezor/trezor-firmware/blob/${revision}/${
```

Comment:
I think that using revision instead of just master is not beneficial to the users and more fragile. see https://github.com/trezor/trezor-suite/issues/4665#issuecomment-1206215039

Reply:
the switch between `core` and `legacy/firmware` should be implemented also here https://github.com/trezor/trezor-suite/blob/fix/4665-changelog-links/packages/suite/src/components/firmware/FirmwareOffer.tsx#L156 (I know, it wasn't part of this issue, but would be nice to fix it.)

I would propose to create a separate constant by model in @trezor/urls for it.

Reply:
I went for a util to determine the URL based on the device and firmware revision. 238ba0ac47ea0577c06a640a71aee903cbf9336c